### DISCUSSION TITLE
Formatter not using environment path

### DISCUSSION THREAD

**Discussion by GNITOAHC** [helix-editor/helix#8582] [Troubleshooting]
`formatter = { command = "clang-format", args = ["--style=file:~/.clang_format"] }` and 
`formatter = { command = "clang-format", args = ["--style=file:/$HOME/.clang_format"] }`
both don't works but if I use the full path i.e. `--style=file:/Users/username/.clang_format` it does works. 
How can I config this `languages.toml` so that this config can be more portable ?
BTW, `--style=file:~/.clang_format` works quite well in my Neovim config so I have no idea why this don't work.

**Comment by kirawi** [comment]
Same thing as https://github.com/helix-editor/helix/issues/5538

**Reply by GNITOAHC** [reply]
Thank's !!!
I use `formatter = { command = "sh", args = [ "-c", "clang-format --style=file:$HOME/.clang_format" ] }` now. But the bad news is that  this can't be sync with the Windows config.
